main.go

```diff
@@ -158,6 +156,48 @@ func connectDB() (*sql.DB, error) {
 	return db, nil
 }
 
+func printCustomer(customerID int, email string, state string) {
+	w := tabwriter.NewWriter(os.Stdout, 0, 0, 1, ' ', tabwriter.Debug)
+	c := &Customer{
+		ID:    int(customerID),
+		Email: email,
+		State: state,
+	}
+	var cArr []*Customer
+	cArr = append(cArr, c)
+	customersPrintHelper(cArr, w)
+	w.Flush()
+}
+
+func printOrder(orderID int, cID int, pID int) {
+	w := tabwriter.NewWriter(os.Stdout, 0, 0, 1, ' ', tabwriter.Debug)
+	o := &Order{
+		ID:         int(orderID),
+		created_at: sql.NullString{String: "", Valid: false},
+		pID:        pID,
+		cID:        cID,
+		sku:        sql.NullString{String: "", Valid: false},
+	}
+	var oArr []*Order
+	oArr = append(oArr, o)
+	orderPrintHelper(oArr, w)
+	w.Flush()
+}
+
+func printProduct(productID int, name string, price float64, sku string) {
+	w := tabwriter.NewWriter(os.Stdout, 0, 0, 1, ' ', tabwriter.Debug)
+	p := &Product{
+		ID:    int(productID),
+		name:  name,
+		price: price,
+		sku:   sql.NullString{String: sku, Valid: true},
+	}
+	var pArr []*Product
+	pArr = append(pArr, p)
+	productPrintHelper(pArr, w)
+	w.Flush()
+}
```

Comment:
[nitpick] these printing helpers are looking good! Here are some minor things that could take it to the next level:

* Update your "print many" helper function to use a [variadic parameter](https://gobyexample.com/variadic-functions), as this just makes calling the function a bit easier since it can support 1 or many objects easily (no need to construct a new array for just 1 object). 
* This function should also be responsible for printing the table header. 
* Have your function accept an [io.Writer](https://pkg.go.dev/io#Writer). This make it easy to change where the/how the function writes its output to (which could be useful for testing). 


[Here is an example](https://go.dev/play/p/8tegNm5H5BX).

Definitely don't take these suggestions as "you need to re-write it in this way", this is mainly just to show alternative approaches. Definitely good enough as-is 👍